contentcuration/contentcuration/utils/cache.py

```diff
@@ -1,6 +1,9 @@
 from django.core.cache import cache
 
 
+DEFERRED_FLAG = "deferred"
```

Comment:
Maybe `__DEFERRED` for the value, just to make it something very unlikely to actually be set on anything?

Reply:
ok